### PR TITLE
First version of ES backup scripts

### PR DESCRIPTION
**Pull request by st**
Connected to https://github.com/SixSq/tasklist/issues/840

### CODE REVIEW

File: backup/src/es/slipstream-es-backup.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SS_HOSTNAME=
```
Comment (konstan) [inline backup/src/es/slipstream-es-backup.conf]: Is this file used? I think you are using `elasticsearch.yml`.

**Comment by konstan** [comment]
Were you able to verify that closing indexes is not needed for performing the snapshot?  Does ES create a valid snapshot from which it's possible to recover in this case?

File: backup/src/es/slipstream-es-backup.conf (hunk 1 above)
Comment (st) [inline backup/src/es/slipstream-es-backup.conf]: Good point, this file is not needed.

**Comment by st** [comment]
Yes, I confirm that the close/open of indices is not needed when performing a snapshot.
I created some events, made a snapshot, deleted one event, and was able to restore the event from snapshot.

**Pull request merged**